web-ng/src/app/components/main-page/main-page.component.ts

```diff
@@ -57,6 +58,12 @@ export class MainPageComponent implements OnInit {
   }
 
   ngOnInit() {
+    // Show title dialog to assign title on a new project.
+    this.subscription.add(
+      this.navigationService
+        .getProjectId$()
+        .subscribe(id => id == ':new' && this.showTitleDialog())
```

Comment:
Please use `NavigationService.LAYER_ID_NEW` instead of `:new`. In the future we should refactor the conditional into NavigationService as well (`isNewProject()`?)